### DISCUSSION TITLE
Enums

### DISCUSSION THREAD

**Discussion by ThiagoDv** [balta-io/2801#6] [Perguntas e Respostas] [answered]
Olá, boa tarde, os Enums podem ser salvos dentro da estrutura da classe principal ou da classe main? Ou assim como as structs somente fora dos escopos da classe principal e da classe main? Agradeço desde já.

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite @thiagodv, você pode ter os enums em um escopo de classe, mas é interessante separa-los em arquivos diferentes.

Mais a frente você vai aprender sobre SOLID, mais precisamente sobre a letra S, que é princípio da responsabilidade única.

Não só uma classe deve fazer uma e somente uma coisa bem feito, como é interessante mantermos uma separação lógica também.

No caso, normalmente crio uma pasta chamada `enums` e coloco todos eles lá.

Ficou claro?

**Reply by andrebaltieri** [reply]
@ThiagoDv estou marcando como concluído, caso tenha mais dúvidas. só postar!
